packages/api/src/services/repository/repository.service.spec.ts

```diff
@@ -16,22 +16,36 @@ describe('RepositoryService', () => {
     repositoryService = new RepositoryService();
   });
 
-  describe('getRepositories', () => {
-    test('should return repositories', async () => {
-      // Mock the response from RepositoryRestService.
-      const rawRepository = {id: 'repo-id', title: 'repo-title', location: ''};
-      const mockResponse = {'': [rawRepository]};
-      mockRepositoryRestService.getRepositories.mockResolvedValue(mockResponse);
-      const expectedResult = new RepositoryList([new Repository(rawRepository as Repository)]);
-
-      // When the service is called to fetch all repositories,
-      const repositories = await repositoryService.getRepositories();
-
-      // Then I expect it to return an instance of RepositoryList,
-      expect(repositories).toBeInstanceOf(RepositoryList);
-      // with all repositories included.
-      expect(repositories).toEqual(expectedResult);
-    });
+  test('should return repositories', async () => {
```

Comment:
Always put the method name in the beginning of the description
`getRepositories should return a RepositoryList` or `getRepositorySizeInfo should return an instance of RepositorySizeInfo`

Reply:
Done